scripts/check-docs-quality.js

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
Might be good to add a shebang and make the file executable to avoid having to invoke it with `node ./check-docs-quality.js`

```
#!/usr/bin/env node
```